test/unit/modules/blocks/utils.js

```diff
@@ -13,268 +13,581 @@
  */
 'use strict';
 
+var rewire = require('rewire');
+var modulesLoader = require('../../../common/modules_loader');
+var BlocksUtils = rewire('../../../../modules/blocks/utils.js');
+
+var viewRow_full_blocks_list = [
+	{
+		b_id: '13068833527549895884',
+		b_height: 3,
+		t_id: '6950874693022090568',
+		t_type: 0,
+	},
+];
+
 describe('blocks/utils', () => {
-	describe('Utils', () => {
-		describe('library', () => {
-			it('should assign logger');
+	var dbStub;
+	var loggerStub;
+	var blockMock;
+	var transactionMock;
+	var accountMock;
+	var blocksUtilsModule;
+	var modulesStub;
+	var __private;
+	var library;
+	var modules;
+
+	beforeEach(() => {
```

Comment:
No beforeEach please :(

Reply:
Actually from my experience, it is the other way round, no `before`.